strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/service/impl/XmlUserService.java

```diff
@@ -98,8 +61,8 @@ public void setUsers(final UsersDto newUsers)
     @Documented
     @Retention(RUNTIME)
     @Qualifier
-    public @interface StrongboxUserServiceQualifier
+    public @interface Xml
```

Comment:
> Yaml ;)

fixed

Reply:
> Instead of adding new annotation for each user service I think it would be better to create one annotation and an Enum type to determine the implementation. See RepositoryIndexCreatorQualifier as an example.

I wouldn't say that it is definitely right way, the thing is that interface shouldn't know its implementations, so from this point it will be wrong to have qualifier annotation defined within interface.

Reply:
I agree here